Add Python declaration of torch._C and torch._C._autograd modules.

xuzhao9 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #46624 Add type annotations to torch._C._distributed_rpc module.
* #46623 Add type annotations for torch._C._distributed_c10d module.
* **#46622 Add Python declaration of torch._C and torch._C._autograd modules.**

Differential Revision: [D24761503](https://our.internmc.facebook.com/intern/diff/D24761503)

xuzhao9 committed:
Add Python declaration of torch._C and torch._C._autograd modules.

[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

walterddr commented on torch/_C/__init__.pyi.in:
```diff
@@ -293,8 +293,14 @@ class Node:
+    def has_default_value(self) -> _bool: ...
 class FunctionSchema:
-    ...
+    arguments: List[Argument]
+    returns: List[Argument]
```
nit: keep ellipsis to indicate that the definition is not complete?
```suggestion
    arguments: List[Argument]
    returns: List[Argument]
    ...
```

xuzhao9 replied:
Right, I've added ellipsis.

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."




[ghstack-poisoned]

xuzhao9 committed:
Update on "Add Python declaration of torch._C and torch._C._autograd modules."


Differential Revision: [D24761503](https://our.internmc.facebook.com/intern/diff/D24761503)

[ghstack-poisoned]
